test/models/team_bot_test.rb

```diff
@@ -209,70 +209,19 @@ def setup
     #assert_not_nil tb2bt
   end
 
-  test "should notify team bots in background when annotation is created or updated" do
```

Comment:
I removed this test because it has no assertions and it seems to have a lot to do with comments, let me know if we should reinstate it.